.github/workflows/flake.yml

```diff
@@ -37,3 +42,23 @@ jobs:
           ignore: 'F821,W503,F841'
           max-line-length: '100'
           args: '--filename */SConscript, SConstruct'
+
+  # Run isort on the tree.
+  # This checks .py files only so misses SConstruct and SConscript files are not checked, rather
+  # for these files check them afterwards.  The output-filter will not be installed for this part
+  # so regressions will be detected but not annotated.
+  isort:
+    name: Isort check
```

Comment:
Does an isort check really belong in `flake.yml`?

Reply:
maybe renaming this flake workflow to something more generic like `cq.yml`? that way we can include all cq related checks in one file.